rules.d/60-persistent-storage.rules.in

```diff
@@ -136,6 +136,12 @@ KERNEL!="sr*|mmcblk[0-9]boot[0-9]", IMPORT{builtin}="blkid"
 LABEL="persistent_storage_blkid_probe_end"
 {% endif %}
 
+# Decrease devlink priority for whole disk of ISO hybrid images, and make the
+# priority for partitions in the image relatively higher. This is for the case
+# that a disk and one of its partition have the same label or so.
+# See issue #28468.
```

Comment:
"Decrease devlink priority for the disk containing an ISO hybrid image and make the priority for partitions in the image higher. When the disk and one of its partitions have the same label or other attributes, we want to link to the partitions. See https://github.com/systemd/systemd/issues/28468."